libs/react-client/src/useChatSession.ts

```diff
@@ -100,6 +100,8 @@ const useChatSession = () => {
 
       const socket = io(uri, {
         path,
+        withCredentials: true,
+        transports: ['websocket'],
```

Comment:
This change works for our use-case, but I believe adding the `transports` option here means that Socket.IO will not fall back to long-polling if a WS connection can't be established.  There are probably still some load balancers/browsers out there that can't handle web-sockets properly.  It might be helpful for these options to become chainlit initialization options that the user can select?